### PR TITLE
add lxd tests

### LINKED ISSUE #46846: [Fluorine] Add Testcase for LXD
### Description of Issue/Question
Add tests for #46231

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?
Add tests for LXD

### What issues does this PR fix or reference?
Closes #46846

### Tests written?

Yes

### Commits signed with GPG?

Yes

### CODE REVIEW

**Review by rallytime — APPROVED** [review]
Very nice.

**Comment by rallytime** [comment]
@gtmanfred Can you fix those documentation errors? Then we can get this in.

File: salt/modules/lxd.py [hunk 1]
```diff
@@ -3213,8 +3213,44 @@ def image_alias_delete(image,
...
+def snapshots_all(container, remote_addr=None, cert=None, key=None, verify_cert=True):
+    '''
+    Get all snapshots for a container
+
+    container :
+        The name of the container to get.
+
+    remote_addr :
+        An URL to a remote Server, you also have to give cert and key if
+        you provide remote_addr and its a TCP Address!
```
Comment (cachedout) [inline salt/modules/lxd.py]: `An URL to a remote server. The 'cert' and 'key' fields must also be provided if 'remote_addr' is defined.`

File: salt/modules/lxd.py [hunk 2]
```diff
@@ -3213,8 +3213,44 @@ def image_alias_delete(image,
...
+            ~/.config/lxc/client.crt
+
+    key :
+        PEM Formatted SSL Key.
+
+        Examples:
+            ~/.config/lxc/client.key
+
+    verify_cert : True
+        Wherever to verify the cert, this is by default True
```
Comment (cachedout) [inline salt/modules/lxd.py]: I don't know what `Wherever to verify the cert...` means in this context. Could we make this more clear, please?

**Pull request merged**